package.json

```diff
@@ -158,7 +158,7 @@
     "leaflet": "0.7.5",
     "less": "2.7.1",
     "less-loader": "2.2.3",
-    "lodash": "3.10.1",
+    "lodash": "4.17.19",
```

Comment:
See @jaroslawjanas, apparently this is all that we needed for the bump to lodash 4 🤣

Reply:
@varunsharma27 Good for you 👍